.kitchen.docker.yml

```diff
@@ -24,6 +24,7 @@ platforms:
     provision_command:
       - /usr/bin/apt-get update
       - /usr/bin/apt-get install apt-transport-https net-tools -y
+      - mkdir /etc/cron.hourly
```

Comment:
Right!   And installing `cron` during the provision stage would fail with upstart errors.   Strangely the `ubuntu-upstart:14.04` docker image didn't have that issue (cron is already installed) - so I went with the path of least resistance here.